### ISSUE TITLE
Implement a better sorting structure

### ISSUE THREAD

**Issue by krantzinator** [krantzinator/tech-conferences-with-childcare#12]
We currently organize by country. This could be further broken down.

Ideas:
Technology/language
Overarching topic (i.e. backend, frontend, ops)
Month of the year
More location breakdown (state/province)

**Comment by scrabill** [comment]
Organizing by topic / theme then country may help the list from getting too cumbersome. 

What do you think?

**Comment by krantzinator** [comment]
Yeah I think that'd offer the most value as compared to other options.
I'm also pondering if we should break this down into separate `.md` files (say, different Topic/Theme lists that are sorted by location, or vice/versa).

**Comment by scrabill** [comment]
I think separate files can work too. They can all link to each other in the main README if that's what we go with.

**Comment by krantzinator** [comment]
I was going through to separate by topic, but it was a bit challenging. Some topics only would have one thing under them (e.g. if "Databases" was its own topic), and some things were hard to fit into a topic (e.g. should MongoDBWorld go under a Databases topic, or lump under...back end? devops? something else?)
For a first pass, I went ahead and just added a Table of Contents so it's easy to see at a glance what countries/continents are represented. We could also have subsections to separate further by state in the U.S., and country in Europe, and province in India/Canada?

Here's the PR #18

**Comment by krantzinator** [comment]
All that to say -- I like the Table of Contents approach versus having separate files for each sort subject.

**Comment by krantzinator** [comment]
also all that to say -- feel free to PR a version with the Table of Contents sorted by topic :)
